### PR TITLE
LPS-198917 Hide fragment duplicates that are added to the draft page

### PR DESCRIPTION
**Pull request by brookedalton**
https://liferay.atlassian.net/browse/LPS-198917

### CODE REVIEW

File: modules/apps/fragment/fragment-web/src/main/java/com/liferay/fragment/web/internal/change/tracking/spi/display/FragmentEntryLinkCTDisplayRender.java [hunk 1]
```diff
@@ -56,7 +56,10 @@ public String getTitle(Locale locale, FragmentEntryLink fragmentEntryLink)
 
 	@Override
 	public boolean isHideable(FragmentEntryLink fragmentEntryLink) {
-		if (fragmentEntryLink.getOriginalFragmentEntryLinkId() == 0) {
+		Layout layout = _layoutLocalService.fetchLayout(
+			fragmentEntryLink.getClassPK());
```
Comment (pei-jung) [inline modules/apps/fragment/fragment-web/src/main/java/com/liferay/fragment/web/internal/change/tracking/spi/display/FragmentEntryLinkCTDisplayRender.java:60]: Use plid (page layout id) to get the layout. Currently, FragmentEntryLink is only used to create the link between fragment and pages, so the classPK happens to be the same as the plid. And unless the table only has the `classPK` column, we should always use `classPK` with `classNameId` to avoid referencing other entities with the same pk.

**Comment by brookedalton** [comment]
ci:test:sf

**Review by pei-jung — APPROVED** [review]

**Comment by pei-jung** [comment]
ci:test:relevant

**Comment by pei-jung** [comment]
ci:test:publications

**Comment by pei-jung** [comment]
@suilin are the failures related?

**Comment by suilin** [comment]
lots of the tests from ci:test:relevant are abord, retrigger the suite again.

**Comment by suilin** [comment]
ci:test:relevant

**Comment by suilin** [comment]
@pei-jung, the failures are not related to current changes, but there is a function test-fix related to the change and I sent it to https://github.com/liferay-publications/liferay-portal/pull/827, should we forward it?

**Comment by pei-jung** [comment]
@suilin why do we need to update the functional test? We didn't change any functional behavior but just hiding the duplicate fragments.

**Comment by suilin** [comment]
@pei-jung, from my understanding I think it won't make any changes for the function tests, but I did the investigation the function test failure only can occur on the current PR. And I updated the behavior change

Steps:
1. Create a Publication
2. Make a change to home page
3. Review Changes
4. View change to Home Page
5. Parents is available to view (LayoutSet)
6. Go to System Settings -> Publications -> Context View Include Production -> Uncheck
7. Parents view is no longer available to view because LayoutSet has not changed in Publication.

Result: Now The Parent is disabled instead of the Parent tab is gone.

**Comment by pei-jung** [comment]
Thanks @suilin. I took another look at this, and this is actually a regression. We're showing more fragments for the page modification. Let's not forward this yet. I'll look into it more tomorrow.

**Comment by suilin** [comment]
@pei-jung, got it, thanks for looking at it.
